e2e/commands_test.go

```diff
@@ -155,7 +138,7 @@ maintainers:
 
 	appData, err := ioutil.ReadFile(tmpDir.Join("tac.dockerapp"))
 	assert.NilError(t, err)
-	assert.Assert(t, golden.Bytes(appData, "init-singlefile.dockerapp"))
```

Comment:
Why changing this `golden.Bytes` to `golden.String` ?

Reply:
This is because when the assertion fails you just have the bytes printed in the screen and you cannot deduce the difference. So since it's a text file I've preferred changing for read/debuggability.